handlers.py

```diff
@@ -2,16 +2,15 @@
 import kubernetes
 import os
 
-def watch_namespace(namespace, **_):
-    namespaces = os.getenv('WATCH_NAMESPACE')
-    if namespaces is None or namespaces == "":
-        return True
+WATCH_NAMESPACE = os.getenv('WATCH_NAMESPACE')
+if WATCH_NAMESPACE is not None:
+    all_namespaces = WATCH_NAMESPACE.split(',')
 
-    all_namespaces = namespaces.split(',')
-    if namespace in all_namespaces:
+def watch_namespace(namespace, **_):
+    if WATCH_NAMESPACE == "" or namespace in all_namespaces:
```

Comment:
If `WATCH_NAMESPACE` env doesn't exist ; `all_namespaces` will be undefined

Reply:
I think we can define `all_namespaces` as empty string

Reply:
Yeah correct, somehow it got removed when i copy pasted from my test server.

Reply:
But again If WATCH_NAMESPACE env doesn't exist ; we can't return true
![image](https://user-images.githubusercontent.com/21688838/107850848-a6b2be80-6e16-11eb-9e3a-9a38a0c7c2c2.png)

Reply:
Ok, in that case if WATCH_NAMESPACE is not set, can we add an  empty string?
```
WATCH_NAMESPACE = os.getenv('WATCH_NAMESPACE', "")
all_namespaces  = WATCH_NAMESPACE.split(',')
```

Reply:
Yes we can use this